src/WordPress_Object/Clarkson_Post_Type.php

```diff
@@ -12,19 +12,13 @@
  * Clarkson Post Type class.
  */
 class Clarkson_Post_Type {
-
-	/**
-	 * @var \WP_Post_Type
-	 */
-	protected $_post_type;
-
 	/**
-	 * @var array
+	 * The WordPress post type object
 	 */
-	protected static $post_types;
+	protected \WP_Post_Type $post_type;
 
 	public function __construct( \WP_Post_Type $post_type ) {
-		$this->_post_type = $post_type;
+		$this->post_type = $post_type;
```

Comment:
Deze wijzigingen kunnen niet zonder dat we naar cc 2.0 gaan. Ik denk niet dat het handig is voor zo'n kleine wijziging. Mogelijk gebruiken classes die de cc base classes extenden de `_user`, `_post_type`, `_taxonomy` properties wel direct.

Reply:
Ja, omdat we deze change eerst op hadden gezet als minimaal 8.1 compatibility heb ik meteen de hele boel aangepast, maar dat gaat nu natuurlijk niet werken.